### DISCUSSION TITLE
Problems viewing items containing passkey

### DISCUSSION THREAD

**Discussion by hchengting** [dani-garcia/vaultwarden#4256] [Q&A] [answered]
### Subject of the issue

Some fields are not displayed when viewing items containing passkey, this problem occurs when the language is set to zh-TW.

### Deployment environment (Generated via diagnostics page)
* Vaultwarden version: v1.30.1
* Web-vault version: v2023.10.0
* OS/Arch: linux/x86_64
* Running within Docker: true (Base: Debian)
* Environment settings overridden: false
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: true
* Server/NTP Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.44.0
* Clients used: Edge & Web Extension for Edge
* Reverse proxy and version: openresty 1.21.4.3-0-focal
* Other relevant information: 

### Steps to reproduce

1. Set the language to zh-TW.
2. View items containing passkey.

### Expected behaviour

All fields should display correctly.
![image](https://github.com/dani-garcia/vaultwarden/assets/74168694/e447843f-ef88-4878-be74-56c2a0bf8470)
![image](https://github.com/dani-garcia/vaultwarden/assets/74168694/5c5e8940-5a53-4738-99ad-9203012380cd)

### Actual behaviour

Some fields are not displayed correctly.
![image](https://github.com/dani-garcia/vaultwarden/assets/74168694/fc94ac62-f685-4fa2-8c02-56825157bae4)
![image](https://github.com/dani-garcia/vaultwarden/assets/74168694/ed668d1d-34be-4d0f-aa48-796ba96ac48e)

**Comment by BlackDex** [comment, marked as answer]
That's not something we can fix i think.
This is all client side, and has nothing to do with the backend server.

**Reply by hchengting** [reply]
Thank you for your reply.
I realized that someone had already asked the same question on the client side.
